server/helpers/paginationHelper.js

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Get pagination meta
+ *
+ * @param   {number}  page current page
+ * @param   {number}  count total data count
+ * @param   {number}  limit total per page
+ *
+ * @return  {object} an object containing offset numberOfPages nextPage and prevPage properties
+ */
+export default function paginationMeta(page, count, limit) {
+    let prevPage;
+    let nextPage;
+    const numberOfPages = Math.ceil(count / limit); // all pages count
+    // check if number of pages is less than the current page number to show next page number
+    if (page < numberOfPages) {
+      nextPage = page + 1;
+    }
+    // show previous page number if page is greater than 1
+    if (page > 1) {
+      prevPage = page - 1;
+    }
+    return {
+      numberOfPages, nextPage, prevPage,
+    };
+  }
```

Comment:
Would be nice to have a blank new line here

Reply:
Done